TDAT: public void test callon consecutive occurrences ( ) throws exception {  <NL> test input = new munge step output < string > ( " test " , string . class ) ;  <NL> test input . set data ( " abcabc " ) ;  <NL> step . connect input ( 0 , test input ) ;  <NL> step . set from ( " abc " ) ;  <NL> step . set to ( " 123 " ) ;  <NL> step . open ( logger ) ;  <NL> step . call ( ) ;  <NL> list < munge step output > results = step . get munge step outputs ( ) ;  <NL> munge step output output = results . get ( 0 ) ;  <NL> string result = ( string ) output . get data ( ) ;  <NL> assert equals ( " 123123 " , result ) ;  <NL>  }  COM: <s> this tests a previous design error where the munge step would </s>